qcengine/programs/psi4.py

```diff
@@ -74,6 +74,10 @@ def compute(self, input_model: "AtomicInput", config: "TaskConfig") -> "AtomicRe
         error_message = None
         compute_success = False
 
+        # Basis must not be None for HF3c
+        old_basis = input_model.model.basis
+        input_model.model.__dict__["basis"] = old_basis or ""
```

Comment:
for my edification, what's the problem with `(none)` exactly? qcf?

Reply:
Psi4, the current JSON drive always attempts to set the basis. Psi4 options can only be strings.